tapir/api/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.urls import path, include
+from rest_framework import routers
+
+# Serializers define the API representation.
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+    TokenVerifyView,
+)
+
+from tapir.api.views import UpcomingShiftAttendanceView, ShareOwnerView, TapirUserView
+
+# Routers provide an easy way of automatically determining the URL conf.
+router = routers.DefaultRouter()
+
+# Wire up our API using automatic URL routing.
+# Additionally, we include login URLs for the browsable API.
+urlpatterns = [
+    path("", include(router.urls)),
+    path(
+        "upcoming_shift_attendance/",
```

Comment:
(nit) Maybe it would make sense to have resource-oriented paths already, in anticipation of more? So `shifts/upcoming_attendances` or even `shifts/attendances/upcoming`?